core-bundle/contao/classes/BackendTemplate.php

```diff
@@ -88,7 +88,7 @@ private function compile()
 			foreach (array_unique($GLOBALS['TL_JAVASCRIPT']) as $javascript)
 			{
 				$options = StringUtil::resolveFlaggedUrl($javascript);
-				$strJavaScripts .= Template::generateScriptTag($this->addStaticUrlTo($javascript), $options->async, $options->mtime);
+				$strJavaScripts .= Template::generateScriptTag($this->addStaticUrlTo($javascript), $options->async, $options->mtime, track: 'dynamic');
```

Comment:
Why does this call use named arguments but the other one does not?

Reply:
Because in the other one it is not necessary. `Template::generateStyleTag` has the following arguments:

* `$href`
* `$media`
* `$mtime`
* `$trag`

Out of these 4 we use all of them here.

`Template::generateScriptTag` on the other hand has the following arguments:

* `$src`
* `$async`
* `$mtime`
* `$hash`
* `$crossorigin`
* `$referrerpolicy`
* `$defer`
* `$track`

Out of these, we only use the first 3 - and then the new `$track` argument.